src/hierarchy.ts

```diff
@@ -275,10 +276,12 @@ export async function group(store?: Store | string, path: string | null = null,
  * @param chunkStore Store or path to directory in file system or name of zip file.
  * @param cacheAttrs If `true` (default), user attributes will be cached for attribute read operations
  *   If False, user attributes are reloaded from the store prior to all attribute read operations.
- *
+ * @param ipfsClient IPFS client
```

Comment:
maybe want to add some additional comments here describing what it is as in the parameters above? (Our PR to them should ideally mesh so well that others wouldn't be able to tell it was an external addition)

so something like IPFS js client which serves as an interface to interact with the IPFS network.